### PR TITLE
Hue binding: remove lightId property

### PR DESCRIPTION
**Pull request by lolodomo**
Fix issue #3965

Also add unit to pollingInterval configuration setting

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java [hunk 1]
```diff
@@ -133,7 +133,6 @@ private synchronized void initializeProperties() {
                 if (vendor != null) {
                     updateProperty(Thing.PROPERTY_VENDOR, vendor);
                 }
-                updateProperty(LIGHT_ID, fullLight.getId());
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java]: Please note that this change will break https://github.com/eclipse/smarthome/issues/3899 as long as https://github.com/eclipse/smarthome/issues/3863#issuecomment-320099215 isn't yet implemented (correct, @afuechsel?).

Comment (afuechsel) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java]: No, don't think so, as UNIQUE_ID is used as representation property, not the light id

Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java]: Ah, ok, and the UNIQUE_ID is something like a Mac address of the bulb?

Comment (lolodomo) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java]: I kept unchanged the set of the LIGHT_UNIQUE_ID property.

**Comment by lolodomo** [comment]
HueBindingConstants in fact defines at the same place the configuration settings and properties.
By the way, I should suppress SERIAL_NUMBER and the binding should use the constant from Thing.

**Comment by lolodomo** [comment]
For the SERIAL_NUMBER, you're right, it seems to be necessary by the bridge handler factory. So I finally propose to keep it !

Regarding the backward compatibily, I have not tested it, I only tested with new things.

So to avoid problems with backward compatibily, I will reduce my PR to only the suppress of the lightId property, which was in fact my initial intention.

**Comment by kaikreuzer** [comment]
>  it seems to be necessary by the bridge handler factory. So I finally propose to keep it !

You are giving up to quickly! I think it should still be removed, we just have to figure out, how to refactor this code.

**Comment by lolodomo** [comment]
I let someone else doing the change with serial number and testing the backward compatibility. I will myself focus on other things.
But I will at least update my PR to cover the lightId property.

**Comment by kaikreuzer** [comment]
ok, fine!

**Comment by lolodomo** [comment]
Updated PR.

**Comment by kaikreuzer** [comment]
Merge conflicts...

**Comment by lolodomo** [comment]
What amount of time just for 2 updated lines ! ;)

**Comment by afuechsel** [comment]
Lgtm

**Pull request merged**